python/tvm/relay/frontend/pytorch.py

```diff
@@ -2094,21 +2094,8 @@ def deform_conv2d(self, inputs, input_types):
 
     def unbind(self, inputs, input_types):
```

Comment:
Remove this method if possible (it is no longer used?)

Reply:
It is used. It is wrapper for pytorch frontend which supports given format of op arguments